### DISCUSSION TITLE
Touch screen button and slider set state

### DISCUSSION THREAD

**Discussion by Jardland** [arendst/Tasmota#15849] [Q&A] [answered]
I can get slider and button to work, but how to use set state in script.

dt [b256:0:0:80:60:2:11:4:2:B1:]
dt [bs0:10:10:200:20:20:1:5:10:]

Sliders~
bs# where # = is slider number 0..15
xp = x position
yp = y position
xa = x size
ys = y size
ne = number of elements
bc = background color
fc = frame color
bc = bar color

Set the state of a button or slider with:
bs# where # = is slider number 0..15
val = 0 or 1 for buttons, 0..100 for sliders

**Comment by gemu2015** [comment, marked as answer]
i will rework the display docs soon

this sets the slider #4 to value 80
dt [b4s80]
